assets/xml/objects/object_f40_obj.xml

```diff
@@ -1,17 +1,25 @@
 ﻿<Root>
+    <!-- 
+        Assets for Stone Tower platforms. These platforms include the cubes moved by pressing the switches in 
+        Stone Tower and two unused platforms.
+    -->
     <File Name="object_f40_obj" Segment="6">
-        <Texture Name="object_f40_obj_Tex_000000" OutName="tex_000000" Format="i8" Width="64" Height="64" Offset="0x0" />
-        <Texture Name="object_f40_obj_Tex_001000" OutName="tex_001000" Format="rgba16" Width="32" Height="64" Offset="0x1000" />
-        <Texture Name="object_f40_obj_Tex_002000" OutName="tex_002000" Format="rgba16" Width="32" Height="32" Offset="0x2000" />
-        <Texture Name="object_f40_obj_Tex_002800" OutName="tex_002800" Format="i4" Width="32" Height="128" Offset="0x2800" />
-        <Texture Name="object_f40_obj_Tex_003000" OutName="tex_003000" Format="i4" Width="64" Height="64" Offset="0x3000" />
-        <DList Name="object_f40_obj_DL_003B00" Offset="0x3B00" />
-        <DList Name="object_f40_obj_DL_003B08" Offset="0x3B08" />
-        <Collision Name="object_f40_obj_Colheader_003E80" Offset="0x3E80" />
-        <DList Name="object_f40_obj_DL_004030" Offset="0x4030" />
-        <DList Name="object_f40_obj_DL_004038" Offset="0x4038" />
-        <Collision Name="object_f40_obj_Colheader_004240" Offset="0x4240" />
-        <DList Name="object_f40_obj_DL_0043D0" Offset="0x43D0" />
-        <Collision Name="object_f40_obj_Colheader_004640" Offset="0x4640" />
+        <Texture Name="gUnusedStoneTowerPlatformsTopTex" OutName="stone_tower_platforms_top" Format="i8" Width="64" Height="64" Offset="0x0" />
+        <Texture Name="gUnusedStoneTowerPlatformsSidesTex" OutName="stone_tower_platforms_sides" Format="rgba16" Width="32" Height="64" Offset="0x1000" />
+        
+        <Texture Name="gStoneTowerBlockBackgroundTex" OutName="stone_tower_block_background" Format="rgba16" Width="32" Height="32" Offset="0x2000" />
+        <Texture Name="gStoneTowerBlockPatternTex" OutName="stone_tower_block_pattern" Format="i4" Width="32" Height="128" Offset="0x2800" />
+        <Texture Name="gStoneTowerBlockSidesTex" OutName="stone_tower_block_sides" Format="i4" Width="64" Height="64" Offset="0x3000" />
+        
+        <DList Name="gStoneTowerEmpty1DL" Offset="0x3B00" />
+        <DList Name="gUnusedStoneTowerVerticallyOscillatingPlatformDL" Offset="0x3B08" />
+        <Collision Name="gUnusedStoneTowerVerticallyOscillatingPlatformCol" Offset="0x3E80" />
+        <DList Name="gStoneTowerEmpty2DL" Offset="0x4030" />
+        
+        <DList Name="gUnusedStoneTowerTempleSquareElevatorDL" Offset="0x4038" />
+        <Collision Name="gUnusedStoneTowerTempleSquareElevatorCol" Offset="0x4240" />
```

Comment:
This is going to be the most nitpicky I think I will ever be, but if you're going to insert an empty line between "groups" of assets, I'd prefer it if `gStoneTowerEmpty2DL` was next to `gUnusedStoneTowerTempleSquareElevatorDL`; in other words, the empty line should appear after `gUnusedStoneTowerVerticallyOscillatingPlatformCol`.

To explain why I'd prefer this, it's helpful to know how the original devs tended to name assets. From OoT3D/MM3D, we can see that they tended to suffix model names with `_modelT` and `_model`, where `_modelT` is used for things that needed to be rendered with transparency, and `_model` is used for things that can be rendered without transparency. So for this particular object, I believe that the two empty DLists you found are actually `_modelT` assets, and since nothing needs to be rendered with transparency, they just left them empty. Thus, I feel like `gStoneTowerEmpty2DL` should *not* be separated from `gUnusedStoneTowerTempleSquareElevatorDL`, since they are likely related.

Unfortunately, all this unused junk was removed from the object in MM3D, so we can only speculate (hence why I said "I believe" earlier). However, you can see this pattern in a bunch of different objects (for example, `object_dblue_object`), so I'm fairly confident that's what's going on here.